executor/distsql.go

```diff
@@ -544,7 +540,7 @@ func (e *IndexLookUpExecutor) Next(ctx context.Context, chk *chunk.Chunk) error
 		start := time.Now()
 		defer func() { e.runtimeStats.Record(time.Now().Sub(start), chk.NumRows()) }()
 	}
-	chk.Reset()
+	chk.GrowAndReset(e.maxChunkSize)
```

Comment:
Before the first iteration on the execution tree, the chunk size is `tidb_init_chunk_size`, when we call `Next()` of the execution tree in the first time, the actual max chunk size for the first returned `Chunk` is 2*`tidb_init_chunk_size`.